base/templates/base/officers.html

```diff
@@ -38,7 +38,11 @@ <h3>
             {{ officer.user.userprofile.get_common_name }}
           {% endif %}
           </span>
-          {{ officer.user.get_username }}@tbp
+		  {% if officer.user.userprofile.alt_officer_email %}
+		  {{ officer.user.userprofile.alt_officer_email }}
+		  {% else %}
+          {{ officer.user.get_username }}@tbp.berkeley.edu
```

Comment:
Leave it as "(at)tbp" to mitigate email scrapers
For the alt_officer_email, we can just use whatever text is inside it (if they choose to do ".edu" that's fine, in case to support ".com" or smth if they change email)

We'll just ask users to remove the domain name (".edu" or ".com") in the "help_text" (see relevant comment)